salt/search/config/etc-logrotate.d-search

```diff
@@ -8,3 +8,12 @@
     copytruncate
 }
 {% endfor %}
+
+/var/log/elasticsearch/elasticsearch.log {
```

Comment:
This is not rotated at all by elasticsearch right?

Reply:
it's rotated, but old ones are not discarded. they are also not compressed, which require features from an 'extras' log4j package

Reply:
I think it's simpler in our case if the application doesn't rotate their logs and just leaves it to logrotate.

Reply:
Yes, just wanted to avoid a race between the two

Reply:
I know what you mean, I think that's what the `copytruncate` option is for:
```
        copytruncate
              Truncate the original log file to zero size in place after creating a copy, instead of moving the old log file and optionally creating a new  one.   It
              can  be  used when some program cannot be told to close its logfile and thus might continue writing (appending) to the previous log file forever. 
```